datahub_sap_hana/column_lineage_schema.py

```diff
@@ -42,9 +43,20 @@ class ColumnField:
     @classmethod
     def from_node(cls, node: Node, schema: str):
         """Creates a ColumnField from a sqlglot node."""
+
+        default_schema = schema
+
+        if isinstance(node.source, expressions.Table):
+            node.source
```

Comment:
This section of code isn't actually doing anything, 
instead you should rename the argument in the function definition and then do this
```suggestion
        schema = default_schema
        if isinstance(node.source, expressions.Table):
            schema = node.source.catalog or node.source.db or default_schema
```